### PR TITLE
Sonobi Video Support

### PR DESCRIPTION
**Pull request by JonGoSonobi**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Added support for video and added ref parameter to Sonobi's creative display rendering

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
apex.prebid@sonobi.com
- [ x] official adapter submission

### CODE REVIEW

**Comment by JonGoSonobi** [comment]
@mjacobsonny Docs PR added. https://github.com/prebid/prebid.github.io/pull/698

**Comment by jongo593** [comment]
@ndhimehta has there been any progress on this? Do you need anything else from Sonobi to get this through?

**Comment by JonGoSonobi** [comment]
@ndhimehta has there been any progress on this? Do you need anything else from Sonobi to get this merged?

File: modules/sonobiBidAdapter.js [hunk 1]
```diff
@@ -76,25 +79,41 @@ export const spec = {
...
+      const videoMediaType = utils.deepAccess(bidRequest, 'mediaTypes.video');
+      const mediaType = bidRequest.mediaType || (videoMediaType ? 'video' : null);
+      const createCreative = _creative(mediaType);
       const bid = bidResponse.slots[slot];
-
       if (bid.sbi_aid && bid.sbi_mouse && bid.sbi_size) {
+        const [
+          width = 1,
+          height = 1
+        ] = bid.sbi_size.split('x');
```
Comment (mkendall07) [inline modules/sonobiBidAdapter.js]: nice!

**Pull request merged**